### PR TITLE
Run pyright on the `tests` directory in CI

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: tests/pytype_test.py [hunk 1]
```diff
@@ -103,10 +103,6 @@ def _get_module_name(filename: str) -> str:
     return ".".join(module_parts).replace(".pyi", "").replace(".__init__", "")
 
 
-def _is_version(path: str, version: str) -> bool:
-    return any("{}{}{}".format(d, os.path.sep, version) in path for d in TYPESHED_SUBDIRS)
-
-
```
Comment (AlexWaygood) [inline tests/pytype_test.py:109]: pyright pointed out that this private function isn't used at all.

**Comment by AlexWaygood** [comment]
Ah, I tested locally in my virtual environment. This might be a bit more complicated. Closing for now.

**Review by hauntsaninja — COMMENTED** [review]
(the benefit of this feels pretty marginal, so I wouldn't try too hard to get it to work)

**Comment by AlexWaygood** [comment]
> (the benefit of this feels pretty marginal, so I wouldn't try too hard to get it to work)

Yeah

**Pull request closed without merging**